### PR TITLE
Refine failure scenario.

### PR DESCRIPTION
**Pull request by yaohaizh**
@akaroml @testforstephen @andxu

### CODE REVIEW

File: src/hotCodeReplace.ts [hunk 1]
```diff
@@ -34,19 +38,23 @@ export function initializeHotCodeReplace(context: vscode.ExtensionContext) {
...
             if (!suppressedReasons.has(customEvent.body.message)) {
-                vscode.window.showInformationMessage(`Hot code replace failed - ${customEvent.body.message}`, NOT_SHOW_AGAIN).then((res) => {
-                    if (res === NOT_SHOW_AGAIN) {
-                        suppressedReasons.add(customEvent.body.message);
-                    }
-                });
+                vscode.window.showInformationMessage(
+                    `Hot code replace failed - ${customEvent.body.message}. Would you like to restart the debug session?`,
+                    YES_BUTTON, NO_BUTTON, NEVER_BUTTON).then((res) => {
+                        if (res === NEVER_BUTTON) {
```
Comment (testforstephen) [inline src/hotCodeReplace.ts]: What does `Never` mean?

Comment (testforstephen) [reply src/hotCodeReplace.ts]: Shouldn't we always show the error message when `HCR` failed? Refer to `Eclipse`, can it be `suppressed`?

Comment (yaohaizh) [reply src/hotCodeReplace.ts]: We are not exactly match eclipse behavior.  Eclipse can configurate on its settings UI>

Comment (yaohaizh) [reply src/hotCodeReplace.ts]: Not show again for NEVER

**Pull request merged**